human.txt

```diff
@@ -108,3 +108,4 @@ yegorov
 yiwenhuang
 zotamias
 zwirektech
+Azlur
```

Comment:
This should be positioned somewhere near the start of this file as according to alphabetical order.